sass/main.scss

```diff
@@ -170,6 +170,69 @@ body {
 		z-index: -4;
 	}
 }
+.news {
```

Comment:
Być może warto byłoby przerzucić sekcje do osobnych plików .scss i tylko importować je w pliku `main.scss`?

Reply:
Zrobiłem tak jak poradziłeś, wrzuciłem sekcje do osobnych plików .scss, a następnie zaimportowałem je do pliku głównego `main.scss`. Mam nadzieję, że zrobiłem to w odpowiedni sposób :)